torch/optim/lbfgs.py

```diff
@@ -442,7 +442,8 @@ def obj_func(x, t, d):
                         return self._directional_evaluate(closure, x, t, d)
 
                     loss, flat_grad, t, ls_func_evals = _strong_wolfe(
-                        obj_func, x_init, t, d, loss, flat_grad, gtd
+                        obj_func, x_init, t, d, loss, flat_grad, gtd,
+                        max_ls=min(25, max_eval - current_evals)
```

Comment:
```suggestion
                        max_ls=max_eval - current_evals
```

The 25 then seems a bit arbitrary? It looks like the 25 is just 20 (max_iter) * 1.25 anyway.